cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -119,17 +125,69 @@ func runSeccompAdvisorStart(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// getSeccompProfileName returns the seccomp profile name associated with the
+// given as parameter traceID.
+// Indeed, a seccomp profile created by seccomp-advisor gadgets has, in its
+// Labels, the trace's id which created it.
+func getSeccompProfileName(traceID string) (string, error) {
+	// seccompprofile does not provide an API to get Get, List, etc. seccomp
+	// profiles, thus we need to make it ourselves.
+	// To be able to retrieve seccompprofile, we need to add them to a scheme.
+	scheme := runtime.NewScheme()
+	seccompprofilev1alpha1.AddToScheme(scheme)
+
+	// Get a manager on seccompprofile.
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: "0", // TCP port can be set to "0" to disable the metrics serving
+		ClientDisableCacheFor: []client.Object{
+			// We need to disable cache otherwise we get the following error message:
+			// the cache is not started, can not read objects
+			// Since this manager will be created each time user interacts with the
+			// CLI the cache will not persist, so there is not really advantages of
+			// using a cache.
+			&seccompprofilev1alpha1.SeccompProfile{},
+		},
+	})
+	if err != nil {
+		return "", fmt.Errorf("Unable to create manager: %s", err)
+	}
+
+	// Get a client on seccompprofile.
+	cli := mgr.GetClient()
+
+	profileList := &seccompprofilev1alpha1.SeccompProfileList{}
+	err = cli.List(context.TODO(), profileList, client.MatchingLabels{utils.GLOBAL_TRACE_ID: traceID})
+	if err != nil {
+		return "", fmt.Errorf("Problem while listing seccomp profiles: %w\n", err)
+	}
```

Comment:
Could you please apply this:
`go-staticcheck`: error strings should not end with punctuation or a newline (ST1005)
`go-staticcheck`: error strings should not be capitalized (ST1005)

Reference https://github.com/golang/go/wiki/CodeReviewComments#error-strings